pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -148,6 +148,7 @@
   company_strongnet: Strongsville School District
   company_TukwilaSchoolDistrict: Tukwila School District
   company_vermilionschools: Vermilion Local School District
+  company_vista-equity-partners: "Vista Equity Partners"
```

Comment:
It would be better to stay consistent but this is an exception because they already have an employee engagement page code.org/employees/vista-equity-partners and I think it'd be a little confusing for them to have two different styles to share with their employees.